perses/rjmc/coordinate_tools.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import simtk.unit as units
+
+def _rotation_matrix(axis, angle):
```

Comment:
Later, we might think of moving to a quaternion representation to considerably simplify things.